lib/active_merchant/billing/gateways/vpos.rb

```diff
@@ -138,7 +139,9 @@ def add_card_data(post, payment)
 
         payload = { card_number: card_number, 'cvv': cvv }.to_json
 
-        post[:card_encrypted_data] = JWE.encrypt(payload, one_time_public_key)
+        encryption_key = @encryption_key || OpenSSL::PKey::RSA.new(one_time_public_key)
```

Comment:
Is there a scenario where the `one_time_public_key` would still be used? I'm assuming this is a failsafe. Is this in place for the time when the switch is being made? Asking for my own knowledge. You mentioned that this would be a breaking change for them so I'm just curious about why the `one_time` is still included here.

Reply:
Yes. In summary, this is my attempt to make the breaking change a bit less breaking. :D